### ISSUE TITLE
Firefox Private Network Extension Reports Offline, IP shows Cloudflare Warp

### ISSUE THREAD

**Issue by hjasond** [mozilla-extensions/secure-proxy#937]
Steps to replicate:

1. Install FPN extension
2. Authenticate FPN extension with Firefox account
3. Click through on-boarding screens
4. Click slider to enable FPN
5. Extension UI refreshes and says offline

Behavior:

FPN extension said "offline" and displays "Firefox Private Network Needs Your Attention!" each time Firefox is launched. However, in this state, all traffic is routed through FPN without any way to disable other than removing the FPN extension. 

Replicated on:

- macOS Catalina 10.15.7 with Firefox 81.0.2 
- macOS Catalina 10.15.7 with Firefox Nightly 83.0a1 (2020-10-13)
- macOS Big Sur Beta 9 with Firefox Nightly 83.0a1 (2020-10-13)

**Comment by hjasond** [comment]
![CleanShot 2020-10-13 at 14 14 38](https://user-images.githubusercontent.com/70112668/95905657-c352c300-0d5e-11eb-9c70-1a6ec051ead1.gif)

GIF walking through the issue showing I am connected to a Cloudflare Warp IP utilizing the FPN without any way to disconnect from the FPN using the extension.

**Comment by bakulf** [comment]
Do you use https-only mode? If yes, this is normal. We are about to push a new release that fixes this bug.

**Comment by hjasond** [comment]
Yes, I do use https-only mode! That explains it, thank you for the information.

**Comment by willc** [comment]
I am not using HTTPS-only mode, and it is still not working for me. Firefox 94.0.1 on Mac 12.0.1

**Comment by aminomancer** [comment]
> 
> 
> I am not using HTTPS-only mode, and it is still not working for me. Firefox 94.0.1 on Mac 12.0.1

Do you use a desktop VPN app or something else that enables a proxy at the system level? I've had a problem with this extension where, if I connect to a VPN or proxy at the system level (for example a desktop program like Nord VPN's) and then connect to FPN with the extension, then disconnect from the system VPN/proxy while FPN is still connected, FPN gets stuck in this state permanently, even after restarting my network interface, even after rebooting my computer. This has happened to me twice. The first time I somehow managed to fix it, but I can't remember how.

So now it's just stuck in this state. I have tried wiping the extension storage but it seems whatever is causing the problem isn't cached with the extension, it's cached with the proxy utils or something. I know whatever I did to fix it involved digging through the source code and deleting *something* but I can't remember for the life of me what it was.
